tools/wpt/browser.py

```diff
@@ -240,8 +241,7 @@ def find_binary_path(self, path=None, channel="nightly"):
         """Looks for the firefox binary in the virtual environment"""
 
         if path is None:
-            # os.getcwd() doesn't include the venv path
-            path = os.path.join(os.getcwd(), "_venv", "browsers", channel)
+            path = self._get_dest(None, channel)
```

Comment:
This will have to change if the `_get_dest` use of `venv_dir()` is actually dead code.